gr-fec/python/fec/bindings/CMakeLists.txt

```diff
@@ -51,6 +51,7 @@ list(APPEND fec_python_files
     )
 
 if(GSL_FOUND)
+add_definitions(-DENABLE_GSL)
```

Comment:
This is an odd place to put it, especially with that indent. Also, the CMake doc says this has been superseded.

I suggest using `target_compile_definitions()` further down. Also, please rename it to `HAVE_GSL` for consistency. The `ENABLE_*` are used as user input options. Checks for existence in the code use `HAVE_`.
```cmake
target_compile_definitions(fec_python PUBLIC -DHAVE_GSL)
```